docs/Getting-started.md

```diff
@@ -0,0 +1,52 @@
+# Start contributing to the Performance plugin
+This guide is intended to give details on how to contribute to the Performance plugin. From setting up the development environment to creating pull requests.
+
+## Prerequisites
+- Node.js
+- Docker
```

Comment:
Can we put links to those two here as well?
* https://nodejs.org/
* https://www.docker.com/products/docker-desktop